README.md

```diff
@@ -132,28 +132,34 @@ the environment variable ``IMAP_DATA_ACCESS_URL`` or within the
 package ``imap_data_access.config["DATA_ACCESS_URL"]``. The default
 is the production server ``https://api.imap-mission.com``.
 
-### Automated use with API Keys
+### Use with API Keys
 
 The default for the CLI is to use the public endpoints.
 To access some unreleased data products and quicklooks, you may
-need elevated permissions. To programmatically get that, you need
+need elevated permissions. To get that, you need
 an API Key, which can be requested from the SDC team.
 
-To use the API Key you can set the `IMAP_API_KEY` environment variable and then use
-the tool as usual.
+There are two ways to provide your API Key:
+
+1. **Environment variable**: Set the `IMAP_API_KEY` environment variable
 
 ```bash
-$ IMAP_API_KEY=<your-api-key>
+$ export IMAP_API_KEY=<your-api-key>
```

Comment:
I still have a slight preference for doing this on the same line like we had it before instead of an export (which is also shell specific).
https://github.com/IMAP-Science-Operations-Center/imap-data-access/pull/286#issuecomment-3498566753

But, I agree this does need to change.